README.md

```diff
@@ -21,10 +21,14 @@ In most cases sensible defaults have been selected.
 > Running using `docker-compose` will ensure that a suitable postgres instance
 > is started. So there is no need to configure the environment variables.
 
-| Environment Variable | Description |
-| -------------------- | ----------- |
-| PORT                 | The port which express listens on (defaults to `4000`). |
-| DB_CONNECTION_URI    | Connection string for database |
+## Environment Variables
+
+| Name | Description | Required |
+| --- | --- | --- |
+| `PORT` | The port which express listens on (defaults to `4000`). | No |
+| `DB_CONNECTION_URI` | Connection string for database | Yes |
+| `EQ_AUTHOR_API_VERSION` | The current Author API version. This is what gets reported on the /status endpoint | No |
+| `NODE_ENV` | Sets the environment the code is running in | Yes |
```

Comment:
Maybe re-arrange so that the required variables are towards the top.
Also I think we can get rid of the GIF files further down in the readme. They are old now and don't really serve much purpose.

Reply:
OK ill make these changes